### ISSUE TITLE
svnversion missing in python:slim ? 

### ISSUE THREAD

**Issue by liaison** [docker-library/python#301]
All of sudden, I cannot install the "pandas" package from the image based on "python:slim"

**Comment by wglambert** [comment]
These are all the packages with "panda" in the name https://packages.debian.org/search?keywords=pandas

In a quick test I'm able to install these just fine, seems however that the "python-pandas" is 539MB

```console
$ docker run --rm -dit --name python python:slim
$ docker exec -it python bash
root@88e5053593d6:/# apt update && apt install python-pandas
Ign:1 http://deb.debian.org/debian stretch InRelease
Get:2 http://security.debian.org/debian-security stretch/updates InRelease [94.3 kB]
Get:3 http://deb.debian.org/debian stretch-updates InRelease [91.0 kB]
Get:4 http://deb.debian.org/debian stretch Release [118 kB]           
Get:5 http://deb.debian.org/debian stretch Release.gpg [2434 B]
Get:6 http://security.debian.org/debian-security stretch/updates/main amd64 Packages [369 kB]
Get:7 http://deb.debian.org/debian stretch-updates/main amd64 Packages [11.0 kB]
Get:8 http://deb.debian.org/debian stretch/main amd64 Packages [7122 kB]       
Fetched 7807 kB in 5s (1544 kB/s)   
Reading package lists... Done
Building dependency tree       
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  binutils blt bzip2 cpp cpp-6 dbus file fontconfig fontconfig-config
  fonts-lyx g++ g++-6 gcc gcc-6 gnome-icon-theme gtk-update-icon-cache
. . .
0 upgraded, 208 newly installed, 0 to remove and 0 not upgraded.
Need to get 157 MB of archives.
After this operation, 539 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```

**Comment by JayH5** [comment]
@liaison the `python:slim` tracks the latest Python version, and Python 3.7.0 was recently released so that's what it is now. `pandas` (version 0.23.1) doesn't have manylinux wheels available for Python 3.7 yet (see [here](https://pypi.org/project/pandas/0.23.1/#files)), so it has to be built from source. The slim image doesn't have any build tools in it, so it's unable to build the source.

You can either use an older Python that does have a wheel uploaded (e.g. `python:3.6-slim`), use a non-slim image (`python:latest`), or install the build tools needed.

**Comment by yosifkit** [comment]
@JayH5's seems to resolve the issue. Closing.
